webui/react/src/components/Markdown.module.scss

```diff
@@ -283,9 +283,9 @@
     margin-left: 0;
   }
   code {
-    background-color: rgba(27, 31, 35, 0.05);
+    background-color: rgb(27 31 35 / 5%);
```

Comment:
Didn't CSS Colors 4 just come out this year? What's the browser support for this?

Reply:
The support is the same as `rgba`. ref: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb

Reply:
I think you're misunderstanding the note there:
> In browsers that support the standard [rgba()](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgba) is an alias for rgb()
Which browsers support the Colors 4 standard?

Reply:
I believe this one https://caniuse.com/css-rrggbbaa